examples/developer-tools/src/controller.ts

```diff
@@ -236,4 +272,73 @@ export class Controller {
     });
     menuEl.replaceChildren(...newItems);
   }
+
+  /** Shows the file upload modal when the user selects that option from the load menu. */
+  private handleLoadFromBlockFactory() {
```

Comment:
Can you change this to `handleLoadFromFile` or `handleLoadFromLegacyBlockFactory` ?